src/nodes.ts

```diff
@@ -22,31 +23,33 @@ export type PropertyKey = Identifier | Literal;
 export type PropertyValue = AssignmentPattern | BindingIdentifier | BindingPattern | FunctionExpression;
 export type StatementListItem = Declaration | Statement;
 
+type Nullable<T> = T | null;
```

Comment:
I don't like this `Nullable<T>` type. It's deceptive. You might think it's something similar to a `Maybe`/`Option`, but it's not. `Nullable<Nullable<T>>` has T+1 inhabitants while `Maybe<Maybe<T>>` has T+2. This is a very important distinction.